src/tooling/piral-cli/src/apps/build-piral.ts

```diff
@@ -160,6 +173,14 @@ export async function buildPiral(baseDir = process.cwd(), options: BuildPiralOpt
       ignored,
     });
 
+    // run post piral build script if provided
+    if (postPiralBuildRelease) {
+      progress(`Running 'postPiralBuildRelease' script ...`);
+      logInfo(`Run: ${postPiralBuildRelease}`);
```

Comment:
We already have the `progress` - I would rather see this as a `log('generalDebug_0003', '...')`.

`logInfo` should be used very sparse.